setup.py

```diff
@@ -28,7 +28,11 @@
 # 'Development Status :: 4 - Beta'
 # 'Development Status :: 5 - Production/Stable'
 release_status = "Development Status :: 3 - Alpha"
-dependencies = ["google-api-core[grpc] >= 1.14.0, < 2.0.0dev"]
+dependencies = [
+    "google-api-core[grpc] >= 1.17.2, < 2.0.0dev",
```

Comment:
```suggestion
    "google-api-core[grpc] >= 1.21.0, < 2.0.0dev",
```

Reply:
can this apply to all libraries? For example, for next library, this should also be `>=1.21.0` or is there some place to reference the minimum version? Thanks!

Reply:
Yes, going forward it will only increase, not decrease. You can reference the setup.py template in the generator https://github.com/googleapis/gapic-generator-python/blob/master/gapic/templates/setup.py.j2#L19 for the current min version.

Reply:
got it, thanks for clarification!